business_rules/engine.py

```diff
@@ -236,6 +236,12 @@ def _build_parameters(method, parameters, extra_parameters):
     else:
         method_params = {}
 
+    if method.params:
```

Comment:
Please, move this into `_build_action_parameters` and do `getattr` on `method` to prevent issues in case action has no `params`